cmd/proxy/actions/cache_miss_fetcher.go

```diff
@@ -43,7 +44,7 @@ func GetProcessCacheMissJob(s storage.Backend, w worker.Worker) worker.Handler {
 			return err
 		}
 
-		if err = s.Save(module, version, v.Mod, zip, v.Info); err != nil {
+		if err = s.Save(context.Background(), module, version, v.Mod, zip, v.Info); err != nil {
```

Comment:
same here RE making the caller pass the context

nit for later

Reply:
I responded above to the comment above on this :)